tests/test_dead.py

```diff
@@ -0,0 +1,19 @@
+import ast
+
+from dead import Visitor
+
+
+def test_getattr_str() -> None:
+    """Tests a call to getattr with a string literal is marked"""
+    method_name = "foo"
+    sample_code = f"""
+class A:
+    def {method_name}():
+        pass
+a = A()
+getattr(a,{method_name})
```

Comment:
this does `getattr(a, foo)` -- you want `getattr(a, 'foo')` -- don't put logic in tests, just write verbatim what's being checked

https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html